src/network/bmproto.py

```diff
@@ -27,7 +26,7 @@
     BMObjectUnwantedStreamError
 )
 from network.proxy import ProxyError
-
+from network import dandelion_ins
```

Comment:
Why in `bmobject` it does `from . import dandelion_ins` and here it does `from network import dandelion_ins`? Why is it inconsistent?

Reply:
Chenged it to `from network import dandelion_ins` everywhere.